drivers/soundwire/mipi_disco.c

```diff
@@ -382,6 +415,8 @@ int sdw_slave_read_prop(struct sdw_slave *slave)
 	sdw_slave_read_dpn(slave, prop->sink_dpn_prop, nval,
 			   prop->sink_ports, "sink");
 
+	sdw_slave_read_lane_mapping(slave);
```

Comment:
do we want to discard errors? The error codes are not handled.

Reply:
Changed to return sdw_slave_read_lane_mapping(slave);